packaging/macos/postinstall-multipass.sh.in

```diff
@@ -16,13 +16,14 @@ mkdir -p ${3}usr/local/bin
 ln -fsv "${3}@CPACK_PACKAGING_INSTALL_PREFIX@/bin/multipass" "${3}usr/local/bin/multipass"
 
 ### Set up symlink for bash competions
-# If HomeBrew installed (only looking for default path)
+# If Homebrew installed (only looking for default path)
+export PATH="${3}opt/homebrew/bin:$PATH"
```

Comment:
This is a ARM Mac specific path. On an Intel Mac, I believe homebrew is located at `/usr/local/`.

Besides, Homebrew specifically instructs the user to add the brew binary to their path in their [install instructions](https://docs.brew.sh/Installation#post-installation-steps). If `brew --prefix` doesn't work, I'm not sure its our responsibility to work around that.

Reply:
Ok, I'm trying to reproduce the issue on my own ARM Mac, and the symlink gets added just fine.